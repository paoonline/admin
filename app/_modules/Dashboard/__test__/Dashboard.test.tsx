import { render } from "@testing-library/react"

import { Providers } from "@/store/provider"
import DashboardModule from "../Dashboard"

describe('DashboardModule', () => {
   it('test', () => {
    const {container} = render(<DashboardModule/>, {wrapper: Providers})
    expect(container).toMatchSnapshot()
   })
})