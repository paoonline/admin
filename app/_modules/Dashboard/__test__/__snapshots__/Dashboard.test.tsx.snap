// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardModule render 1`] = `
<div>
  <div
    class="MuiBox-root css-8atqhb"
  >
    <div
      class="MuiBox-root css-1gsv261"
    >
      <div
        class="MuiTabs-root css-1ujnqem-MuiTabs-root"
      >
        <div
          class="MuiTabs-scroller MuiTabs-fixed css-jpln7h-MuiTabs-scroller"
          style="overflow: hidden; margin-bottom: 0px;"
        >
          <div
            aria-label="basic tabs example"
            class="MuiTabs-flexContainer css-heg063-MuiTabs-flexContainer"
            role="tablist"
          >
            <button
              aria-selected="true"
              class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary Mui-selected css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
              role="tab"
              tabindex="0"
              type="button"
            >
              <span>
                Dashboard
              </span>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
          <span
            class="MuiTabs-indicator css-1aquho2-MuiTabs-indicator"
            style="left: 0px; width: 0px;"
          />
        </div>
      </div>
    </div>
    <div
      aria-labelledby="simple-tab-0"
      class="MuiBox-root css-0"
      id="simple-tabpanel-0"
      role="tabpanel"
    >
      <div
        class="MuiBox-root css-19kzrtu"
      >
        <div
          class="MuiBox-root css-v2kfba"
        >
          <div
            class="w-full MuiBox-root css-17tcgb"
          >
            <h5>
              Machine display
            </h5>
            <div
              class="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-sizeMedium MuiInputLabel-outlined css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
                data-shrink="false"
                for="search-table"
                id="search-table-label"
              >
                Search
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
              >
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                  id="search-table"
                  type="text"
                  value=""
                />
                <fieldset
                  aria-hidden="true"
                  class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="css-yjsfm1"
                  >
                    <span>
                      Search
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
          </div>
          <div
            class="w-full MuiBox-root css-29r2by"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              Create
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
          <div
            aria-colcount="6"
            aria-multiselectable="false"
            aria-rowcount="2"
            class="MuiDataGrid-root MuiDataGrid-root--densityStandard MuiDataGrid-withBorderColor css-az8st9-MuiDataGrid-root"
            role="grid"
          >
            <div
              class="MuiDataGrid-main css-204u17-MuiDataGrid-main"
            >
              <div
                class="MuiDataGrid-columnHeaders MuiDataGrid-withBorderColor css-1iyq7zh-MuiDataGrid-columnHeaders"
                role="presentation"
                style="min-height: 56px; max-height: 56px; line-height: 56px;"
              >
                <div
                  class="MuiDataGrid-columnHeadersInner MuiDataGrid-columnHeadersInner--scrollable css-gl260s-MuiDataGrid-columnHeadersInner"
                  role="rowgroup"
                  style="transform: translate3d(0px, 0px, 0px);"
                >
                  <div
                    aria-rowindex="1"
                    class="css-yrdy0g-MuiDataGrid-columnHeaderRow"
                    role="row"
                  >
                    <div
                      aria-colindex="1"
                      aria-label="ID"
                      aria-sort="none"
                      class="MuiDataGrid-columnHeader MuiDataGrid-columnHeader--sortable MuiDataGrid-withBorderColor"
                      data-field="id"
                      role="columnheader"
                      style="height: 56px; width: 50px; min-width: 50px; max-width: 50px;"
                      tabindex="0"
                    >
                      <div
                        class="MuiDataGrid-columnHeaderDraggableContainer"
                        draggable="false"
                        role="presentation"
                      >
                        <div
                          class="MuiDataGrid-columnHeaderTitleContainer"
                          role="presentation"
                        >
                          <div
                            class="MuiDataGrid-columnHeaderTitleContainerContent"
                          >
                            <div
                              aria-label=""
                              class="MuiDataGrid-columnHeaderTitle css-t89xny-MuiDataGrid-columnHeaderTitle"
                              data-mui-internal-clone-element="true"
                            >
                              ID
                            </div>
                          </div>
                          <div
                            class="MuiDataGrid-iconButtonContainer css-ltf0zy-MuiDataGrid-iconButtonContainer"
                          >
                            <button
                              aria-label="Sort"
                              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                              tabindex="-1"
                              title="Sort"
                              type="button"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall MuiDataGrid-sortIcon css-ptiqhd-MuiSvgIcon-root"
                                data-testid="ArrowUpwardIcon"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M4 12l1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
                                />
                              </svg>
                              <span
                                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                              />
                            </button>
                          </div>
                        </div>
                        <div
                          class="MuiDataGrid-menuIcon"
                        >
                          <button
                            aria-expanded="false"
                            aria-haspopup="menu"
                            aria-label="Menu"
                            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall MuiDataGrid-menuIconButton css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                            data-mui-internal-clone-element="true"
                            id=":r5:"
                            tabindex="-1"
                            type="button"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                              data-testid="TripleDotsVerticalIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                              />
                            </svg>
                            <span
                              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                            />
                          </button>
                        </div>
                      </div>
                      <div
                        class="MuiDataGrid-columnSeparator MuiDataGrid-columnSeparator--sideRight"
                        style="min-height: 56px; opacity: 1;"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiDataGrid-iconSeparator css-i4bv87-MuiSvgIcon-root"
                          data-testid="SeparatorIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M11 19V5h2v14z"
                          />
                        </svg>
                      </div>
                    </div>
                    <div
                      aria-colindex="2"
                      aria-label="Uptime"
                      aria-sort="none"
                      class="MuiDataGrid-columnHeader MuiDataGrid-columnHeader--sortable MuiDataGrid-withBorderColor"
                      data-field="uptime"
                      role="columnheader"
                      style="height: 56px; width: 130px; min-width: 130px; max-width: 130px;"
                      tabindex="-1"
                    >
                      <div
                        class="MuiDataGrid-columnHeaderDraggableContainer"
                        draggable="false"
                        role="presentation"
                      >
                        <div
                          class="MuiDataGrid-columnHeaderTitleContainer"
                          role="presentation"
                        >
                          <div
                            class="MuiDataGrid-columnHeaderTitleContainerContent"
                          >
                            <div
                              aria-label=""
                              class="MuiDataGrid-columnHeaderTitle css-t89xny-MuiDataGrid-columnHeaderTitle"
                              data-mui-internal-clone-element="true"
                            >
                              Uptime
                            </div>
                          </div>
                          <div
                            class="MuiDataGrid-iconButtonContainer css-ltf0zy-MuiDataGrid-iconButtonContainer"
                          >
                            <button
                              aria-label="Sort"
                              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                              tabindex="-1"
                              title="Sort"
                              type="button"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall MuiDataGrid-sortIcon css-ptiqhd-MuiSvgIcon-root"
                                data-testid="ArrowUpwardIcon"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M4 12l1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
                                />
                              </svg>
                              <span
                                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                              />
                            </button>
                          </div>
                        </div>
                        <div
                          class="MuiDataGrid-menuIcon"
                        >
                          <button
                            aria-expanded="false"
                            aria-haspopup="menu"
                            aria-label="Menu"
                            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall MuiDataGrid-menuIconButton css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                            data-mui-internal-clone-element="true"
                            id=":rb:"
                            tabindex="-1"
                            type="button"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                              data-testid="TripleDotsVerticalIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                              />
                            </svg>
                            <span
                              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                            />
                          </button>
                        </div>
                      </div>
                      <div
                        class="MuiDataGrid-columnSeparator MuiDataGrid-columnSeparator--sideRight"
                        style="min-height: 56px; opacity: 1;"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiDataGrid-iconSeparator css-i4bv87-MuiSvgIcon-root"
                          data-testid="SeparatorIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M11 19V5h2v14z"
                          />
                        </svg>
                      </div>
                    </div>
                    <div
                      aria-colindex="3"
                      aria-label="Current sales"
                      aria-sort="none"
                      class="MuiDataGrid-columnHeader MuiDataGrid-columnHeader--sortable MuiDataGrid-withBorderColor"
                      data-field="current_sales"
                      role="columnheader"
                      style="height: 56px; width: 130px; min-width: 130px; max-width: 130px;"
                      tabindex="-1"
                    >
                      <div
                        class="MuiDataGrid-columnHeaderDraggableContainer"
                        draggable="false"
                        role="presentation"
                      >
                        <div
                          class="MuiDataGrid-columnHeaderTitleContainer"
                          role="presentation"
                        >
                          <div
                            class="MuiDataGrid-columnHeaderTitleContainerContent"
                          >
                            <div
                              aria-label=""
                              class="MuiDataGrid-columnHeaderTitle css-t89xny-MuiDataGrid-columnHeaderTitle"
                              data-mui-internal-clone-element="true"
                            >
                              Current sales
                            </div>
                          </div>
                          <div
                            class="MuiDataGrid-iconButtonContainer css-ltf0zy-MuiDataGrid-iconButtonContainer"
                          >
                            <button
                              aria-label="Sort"
                              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                              tabindex="-1"
                              title="Sort"
                              type="button"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall MuiDataGrid-sortIcon css-ptiqhd-MuiSvgIcon-root"
                                data-testid="ArrowUpwardIcon"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M4 12l1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
                                />
                              </svg>
                              <span
                                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                              />
                            </button>
                          </div>
                        </div>
                        <div
                          class="MuiDataGrid-menuIcon"
                        >
                          <button
                            aria-expanded="false"
                            aria-haspopup="menu"
                            aria-label="Menu"
                            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall MuiDataGrid-menuIconButton css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
                            data-mui-internal-clone-element="true"
                            id=":rh:"
                            tabindex="-1"
                            type="button"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                              data-testid="TripleDotsVerticalIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                              />
                            </svg>
                            <span
                              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                            />
                          </button>
                        </div>
                      </div>
                      <div
                        class="MuiDataGrid-columnSeparator MuiDataGrid-columnSeparator--sideRight"
                        style="min-height: 56px; opacity: 1;"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiDataGrid-iconSeparator css-i4bv87-MuiSvgIcon-root"
                          data-testid="SeparatorIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M11 19V5h2v14z"
                          />
                        </svg>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiDataGrid-virtualScroller css-qvtrhg-MuiDataGrid-virtualScroller"
                role="presentation"
                style="overflow-x: hidden;"
              >
                <div
                  class="MuiDataGrid-virtualScrollerContent MuiDataGrid-virtualScrollerContent--overflowed css-1kwdphh-MuiDataGrid-virtualScrollerContent"
                  role="presentation"
                  style="width: auto; height: 52px; min-height: auto;"
                >
                  <div
                    class="MuiDataGrid-virtualScrollerRenderZone css-s1v7zr-MuiDataGrid-virtualScrollerRenderZone"
                    role="rowgroup"
                    style="transform: translate3d(0px, 0px, 0px);"
                  >
                    <div
                      aria-rowindex="2"
                      aria-selected="false"
                      class="MuiDataGrid-row MuiDataGrid-row--lastVisible"
                      data-id=""
                      data-rowindex="0"
                      role="row"
                      style="max-height: 52px; min-height: 52px;"
                    >
                      <div
                        aria-colindex="1"
                        aria-colspan="1"
                        class="MuiDataGrid-cell MuiDataGrid-cell--textLeft MuiDataGrid-withBorderColor"
                        data-colindex="0"
                        data-field="id"
                        role="cell"
                        style="min-width: 50px; max-width: 50px; min-height: 52px; max-height: 52px;"
                        tabindex="-1"
                      >
                        <div
                          class="MuiDataGrid-cellContent"
                          role="presentation"
                          title=""
                        />
                      </div>
                      <div
                        aria-colindex="2"
                        aria-colspan="1"
                        class="MuiDataGrid-cell MuiDataGrid-cell--textLeft MuiDataGrid-withBorderColor"
                        data-colindex="1"
                        data-field="uptime"
                        role="cell"
                        style="min-width: 130px; max-width: 130px; min-height: 52px; max-height: 52px;"
                        tabindex="-1"
                      >
                        <div
                          class="MuiDataGrid-cellContent"
                          role="presentation"
                          title=""
                        />
                      </div>
                      <div
                        aria-colindex="3"
                        aria-colspan="1"
                        class="MuiDataGrid-cell MuiDataGrid-cell--textLeft MuiDataGrid-withBorderColor"
                        data-colindex="2"
                        data-field="current_sales"
                        role="cell"
                        style="min-width: 130px; max-width: 130px; min-height: 52px; max-height: 52px;"
                        tabindex="-1"
                      >
                        <div
                          class="MuiDataGrid-cellContent"
                          role="presentation"
                          title=""
                        />
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiDataGrid-footerContainer MuiDataGrid-withBorderColor css-wop1k0-MuiDataGrid-footerContainer"
            >
              <div />
              <div
                class="MuiTablePagination-root css-rtrcn9-MuiTablePagination-root"
              >
                <div
                  class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular MuiTablePagination-toolbar css-78c6dr-MuiToolbar-root-MuiTablePagination-toolbar"
                >
                  <div
                    class="MuiTablePagination-spacer css-1psng7p-MuiTablePagination-spacer"
                  />
                  <p
                    class="MuiTablePagination-selectLabel css-pdct74-MuiTablePagination-selectLabel"
                    id=":r2:"
                  >
                    Rows per page:
                  </p>
                  <div
                    class="MuiInputBase-root MuiInputBase-colorPrimary MuiTablePagination-input css-16c50h-MuiInputBase-root-MuiTablePagination-select"
                  >
                    <div
                      aria-controls=":r3:"
                      aria-expanded="false"
                      aria-haspopup="listbox"
                      aria-labelledby=":r2: :r1:"
                      class="MuiSelect-select MuiTablePagination-select MuiSelect-standard MuiInputBase-input css-194a1fa-MuiSelect-select-MuiInputBase-input"
                      id=":r1:"
                      role="combobox"
                      tabindex="0"
                    >
                      10
                    </div>
                    <input
                      aria-hidden="true"
                      aria-invalid="false"
                      class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
                      tabindex="-1"
                      value="10"
                    />
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiTablePagination-selectIcon MuiSelect-iconStandard css-pqjvzy-MuiSvgIcon-root-MuiSelect-icon"
                      data-testid="ArrowDropDownIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M7 10l5 5 5-5z"
                      />
                    </svg>
                  </div>
                  <p
                    class="MuiTablePagination-displayedRows css-levciy-MuiTablePagination-displayedRows"
                  >
                    1–1 of 1
                  </p>
                  <div
                    class="MuiTablePagination-actions"
                  >
                    <button
                      aria-label="Go to previous page"
                      class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-colorInherit MuiIconButton-sizeMedium css-zylse7-MuiButtonBase-root-MuiIconButton-root"
                      disabled=""
                      tabindex="-1"
                      title="Go to previous page"
                      type="button"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                        data-testid="KeyboardArrowLeftIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M15.41 16.09l-4.58-4.59 4.58-4.59L14 5.5l-6 6 6 6z"
                        />
                      </svg>
                    </button>
                    <button
                      aria-label="Go to next page"
                      class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-colorInherit MuiIconButton-sizeMedium css-zylse7-MuiButtonBase-root-MuiIconButton-root"
                      disabled=""
                      tabindex="-1"
                      title="Go to next page"
                      type="button"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                        data-testid="KeyboardArrowRightIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M8.59 16.34l4.58-4.59-4.58-4.59L10 5.75l6 6-6 6z"
                        />
                      </svg>
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      aria-labelledby="simple-tab-1"
      class="MuiBox-root css-0"
      hidden=""
      id="simple-tabpanel-1"
      role="tabpanel"
    />
  </div>
</div>
`;
